### PR TITLE
HtmlEditor "Output Format" demo: add the Prettier formatting

### PR DESCRIPTION
**Pull request by marcbambalan**

### CODE REVIEW

**Comment by ksercs** [comment]
@Madobyte  You should update screenshot etalons and package-lock to make tests green

File: NetCoreDemos/Views/HtmlEditor/OutputFormats.cshtml [hunk 1]
```diff
@@ -67,17 +69,33 @@
...
     function selectionChanged(e) {
         var editorInstance = $("#html_editor").dxHtmlEditor("instance");
-        editorInstance.option("valueType", e.addedItems[0].text.toLowerCase());
-        $(".value-content").text(editorInstance.option("value"));
+
+        var valueType = e.addedItems[0].text.toLowerCase();
+        editorInstance.option({ valueType });
+        var value = editorInstance.option("value");
+
+        updateOutput(value, valueType)
```
Comment (ksercs) [inline NetCoreDemos/Views/HtmlEditor/OutputFormats.cshtml]:
```suggestion
        updateOutput(value, valueType);
```

File: JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/index.js [hunk 2]
```diff
@@ -1,4 +1,9 @@
 $(() => {
+  const updateOutput = (value, valueType) => {
```
Comment (ksercs) [inline JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/index.js]: Maybe it's better to name it `formatOutput`? Actual name is too common for me.
Please refactor in for all approaches

Comment (marcbambalan) [reply JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/index.js]: How about `prettierFormat` instead?

**Review by ksercs — APPROVED** [review]

**Pull request merged**